projects/storefrontlib/src/cms-components/checkout/components/review-submit/review-submit.component.ts

```diff
@@ -180,16 +203,28 @@ export class ReviewSubmitComponent {
     );
   }
 
-  getPoNumberCard(poNumber: string, costCenter: string): Observable<Card> {
+  getPoNumberCard(poNumber: string): Observable<Card> {
     return combineLatest([
-      this.translation.translate('checkoutProgress.poNumber'),
-      this.translation.translate('checkoutPO.costCenter'),
+      this.translation.translate('checkoutReview.poNumber'),
+      this.translation.translate('checkoutPO.noPoNumber'),
     ]).pipe(
-      map(([textTitle, textCostCenter]) => {
+      map(([textTitle, noneTextTitle]) => {
+        return {
+          title: textTitle,
+          textBold: poNumber ? poNumber : noneTextTitle,
+        };
+      })
+    );
+  }
+
+  getPaymentTypeCard(paymentType: string): Observable<Card> {
+    return combineLatest([
+      this.translation.translate('checkoutProgress.methodOfPayment'),
+    ]).pipe(
+      map(([textTitle]) => {
         return {
           title: textTitle,
-          textBold: poNumber,
-          text: [costCenter ? textCostCenter + ': ' + costCenter : ''],
+          textBold: paymentType.charAt(0) + paymentType.slice(1).toLowerCase(),
```

Comment:
lets change this to be taken from translation, because I forgot we do not want to display Card, but Credit Card. Unless, Card is fine.

Account is fine as it is, but Card or Credit Card (mock says Credit Card though)

Reply:
afterwards, make sure to wrap the paymentType in Boolean as I just notice there was an error in console